psi4/src/psi4/libfock/jk.cc

```diff
@@ -74,9 +74,9 @@ std::shared_ptr<JK> JK::build_JK(std::shared_ptr<BasisSet> primary, std::shared_
 
         return std::shared_ptr<JK>(jk);
 
```

Comment:
What happens if someone hits this constructor with the "DF" type? Should we add a DiskDFJK fallback for that here.

Reply:
There will be a throw for unknown `SCF_TYPE`.  The only way to avoid this is to use the correct constructor that maps `DF` to the correct type.  I think the throw might be better than a fallback, since it would be the programmers fault and not the user.